controllers/courseController.js

```diff
@@ -0,0 +1,50 @@
+const UnitOfWork = require('../data/UnitOfWork');
+
+const uow = new UnitOfWork();
```

Comment:
Unless it's convention or industry standard it's best to avoid abbreviations in variable names. you could just type `unitOfWork` and then with autocomplete it's not a big deal. However you can give it a more friendly name if you want 🤷‍♂